src/main/java/seedu/address/logic/parser/ViewCommandParser.java

```diff
@@ -97,6 +109,23 @@ public ViewCommand parse(String args) throws ParseException {
         case VIEW_DEADLINE_LIST:
             return new ViewTaskListCommand(VIEW_DEADLINE_LIST);
 
+        case VIEW_CONSULTATION_LIST:
```

Comment:
Looks great, maybe we need to extract some cases out later, to reduce the code to beneath 30loc.

Reply:
Good idea!